taxi/views.py

```diff
@@ -34,12 +35,52 @@
     paginate_by = 5
 
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Manufacturer
+    fields = "__all__"
```

Comment:
Using `fields = "__all__"` in `CreateView` and `UpdateView` might expose all fields of the model to the form, including those that should not be edited by users (e.g., auto-generated fields). Consider specifying only the fields that should be editable by users to ensure better control over the form data.